network/bridge_endpointclient_linux.go

```diff
@@ -126,30 +127,30 @@ func (client *LinuxBridgeEndpointClient) DeleteEndpointRules(ep *endpoint) {
 	for _, ipAddr := range ep.IPAddresses {
 		if ipAddr.IP.To4() != nil {
 			// Delete ARP reply rule.
-			log.Printf("[net] Deleting ARP reply rule for IP address %v on %v.", ipAddr.String(), ep.Id)
+			log.Logger.Info("[net] Deleting ARP reply rule for IP address on", zap.String("address", ipAddr.String()), zap.String("Id", ep.Id), zap.String("component", "net"))
```

Comment:
I suggest keeping the keys consistent. Everything else is lowercase, so "id" should be as well.
```suggestion
			log.Logger.Info("[net] Deleting ARP reply rule for IP address on", zap.String("address", ipAddr.String()), zap.String("id", ep.Id), zap.String("component", "net"))
```